cli/cmd/vuln_container_list_assessments.go

```diff
@@ -65,30 +66,38 @@ filter on containers with vulnerabilities that have fixes available.`,
 						StartTime: &start,
 						EndTime:   &end,
 					}
-					msg = fmt.Sprintf("Fetching assessments in date range %s - %s", start, end)
+					timeRangeMsg = fmt.Sprintf(" in date range (%s to %s)", start, end)
 				}
 
 				filter.TimeFilter = &api.TimeFilter{
 					StartTime: &before,
 					EndTime:   &now,
 				}
```

Comment:
@dmurray-lacework I just noticed that the filter never worked. We always use 7 days here, we just ignore the start and end time 😭